pkg/commands/git_commands/commit.go

```diff
@@ -100,11 +116,15 @@ func (self *CommitCommands) CommitCmdObj(summary string, description string) osc
 }
 
 func (self *CommitCommands) RewordLastCommitInEditorCmdObj() oscommands.ICmdObj {
-	return self.cmd.New(NewGitCmd("commit").Arg("--allow-empty", "--amend", "--only").ToArgv())
+
+	return self.cmd.New(NewGitCmd("commit").
+		// TODO: how to decide if we should add --no-verify if we're using the editor?
```

Comment:
I guess the best we can do is to decide based on the existing subject and hope they don't change it in the editor. Either that, or just never pass `--no-verify` in this case.

Reply:
I think it's safer not to run it, than to run it based on luck.